tests/shared/test_data.py

```diff
@@ -130,6 +130,7 @@ def test_get_core_nlu_files(project):
             ],
         ),
         ("wit", ["data/examples/wit/demo-flights.json"]),
+        ("watson", ["data/examples/watson/demo_watson_v2.json"]),
```

Comment:
this one fails as you'd have to convert it first to a Rasa Open Source compatible format
```suggestion
```